Packs/Box/Integrations/README.md

```diff
@@ -0,0 +1,518 @@
+Manage Box users
+## Configure Box on Cortex XSOAR
```

Comment:
missing here the configuration steps
you can see them in the detailed description

Reply:
Done.